packages/twenty-front/src/modules/object-record/hooks/useDeleteManyRecords.ts

```diff
@@ -101,33 +98,28 @@ export const useDeleteManyRecords = ({
 
           const computedOptimisticRecord = {
             ...cachedRecord,
-            ...{ id: cachedRecord.id, deletedAt: currentTimestamp },
-            ...{ __typename: capitalize(objectMetadataItem.nameSingular) },
+            deletedAt: currentTimestamp,
+            __typename: getObjectTypename(objectMetadataItem.nameSingular),
           };
-
           const optimisticRecordNode = getRecordNodeFromRecord<ObjectRecord>({
             record: computedOptimisticRecord,
             objectMetadataItem,
             objectMetadataItems,
             computeReferences: false,
           });
 
-          if (
-            !isDefined(optimisticRecordNode) ||
-            !isDefined(cachedRecordNode)
-          ) {
-            return;
-          }
-
-          updateRecordFromCache({
-            objectMetadataItems,
-            objectMetadataItem,
-            cache: apolloClient.cache,
-            record: computedOptimisticRecord,
-          });
+          if (isDefined(optimisticRecordNode) && isDefined(cachedRecordNode)) {
```

Comment:
not necessary but OK

Reply:
I can feel you, while going for second pass will favorise early returns 👍